xlr/converters/src/ts-to-xlr.ts

```diff
@@ -191,6 +202,13 @@ export class TsConverter {
           resultingNode = this.tsLiteralToType(variable.initializer);
         }
 
+        // If initializer type is a reference to a function and not a concrete value
+        // we need to update the name to be the name of the exporting variable
+        // not the name of the identifier its aliasing
+        if (ts.isFunctionLike(typeNode)) {
```

Comment:
Instead of doing ts magic to see if the aliased node is a function, can we just check the type that comes back in `resultingNode`?

Reply:
`resultingNode` does not report as being `FunctionLike` as it returns `NodeType` which is an XLR type, not a TS type

Reply:
Sorry, let me clarify. You can check the type of the XLR node that is returned to see if it is a function type node.